paulshryock/paul-shryock
# [#11]: Bundle JavaScript modules with esbuild

## Summary
Bundle JavaScript modules to support browsers without support for modules.

## Testing
- Run `npm start`
- Confirm that JavaScript modules are bundled

## Issue(s)

Closes #11

## Changelog

### Added
- Bundle JavaScript modules. [#11]

## Release Checklist
